src/backend/core/core/settings.py

```diff
@@ -57,9 +55,9 @@
 ]
 
 MIDDLEWARE = [
-    "corsheaders.middleware.CorsMiddleware",
     "django.middleware.security.SecurityMiddleware",
     "django.contrib.sessions.middleware.SessionMiddleware",
+    "corsheaders.middleware.CorsMiddleware",
```

Comment:
Is this transitions safe? Quoting from [documentation](https://pypi.org/project/django-cors-headers/) "CorsMiddleware should be placed as high as possible, especially before any middleware that can generate responses"

Reply:
> Is this transitions safe? Quoting from [documentation](https://pypi.org/project/django-cors-headers/) "CorsMiddleware should be placed as high as possible, especially before any middleware that can generate responses"

Good point, but the question is whether we really want to put it in front of something called `SecurityMiddleware`. Check [this](https://docs.djangoproject.com/en/4.1/ref/middleware/#middleware-ordering) out. It was mainly a test, because I had some CORS issues, but i'd rather stick to the part of [documentation](https://pypi.org/project/django-cors-headers/) where they provide an example:
```python
MIDDLEWARE = [
    ...,
    "corsheaders.middleware.CorsMiddleware",
    "django.middleware.common.CommonMiddleware",
    ...,
]
```
and put it right in front of `CommonMiddleware`.

What do you think?

Reply:
Agreed.